backend/apps/owasp/management/commands/owasp_sync_board_members.py

```
@@ -0,0 +1,178 @@
+            type=int,
+            help="Specific year to sync board members for",
+            required=True,
+        )
+
+    def handle(self, *args: Any, **options: Any) -> None:
+        """Handle command execution."""
+        year = options["year"]
+
+        self.stdout.write("Fetching OWASP Board of Directors data...")
+        try:
+            response = requests.get(BOARD_HISTORY_URL)
```

Ruff (0.13.2)

Probable use of `requests` call without timeout

(S113)